pkg/kubelet/kubelet_pods.go

```diff
@@ -1170,6 +1179,12 @@ func (kl *Kubelet) HandlePodCleanups() error {
 		klog.ErrorS(err, "Failed cleaning up orphaned pod directories")
 	}
 
+	// Remove orphaned pod user namespace allocations (if any).
+	klog.V(3).InfoS("Clean up orphaned pod user namespace allocations")
+	if err = kl.usernsManager.CleanupOrphanedPodUsernsAllocations(allPods, runningRuntimePods); err != nil {
```

Comment:
should this move before we clean up the orphaned pod directories where it appears this manager pulls its state?

so flow is

- cleanupOrphanedPodUsernsAllocation
- cleanupOrphanedPodDirs

Reply:
related

its not clear to me in the present PR where the mappings file is actually deleted, i expected to see a spot where the file is deleted, if i missed it, can you point me to where I missed it?  the cleanupOrphanedPodDirs operation above is assuming that once all sub-directories are removed the pod dir is actually empty, and it looks like it may not actually be empty.

Reply:
we could add that but it seemed not necessary as `cleanupOrphanedPodDirs` already deletes al the files/directories under the pod directory

Reply:
We updated to do what @derekwaynecarr suggested, though

Reply:
@giuseppe changed the flow and added the delete of the file too